package.json

```diff
@@ -42,6 +44,7 @@
     "react-icons": "^2.2.7",
     "react-redux": "^5.0.6",
     "react-router-dom": "^4.2.2",
-    "redux": "^3.7.2"
+    "redux": "^3.7.2",
+    "webpack-sass-loaders": "^1.0.0"
```

Comment:
are we sure this is the right package ?
cause in the webpack.config file you are saying it's sass-loader not wewbpack-sass-loaders

Reply:
it works